gravity.sh

```diff
@@ -317,6 +344,45 @@ gravity_CheckDNSResolutionAvailable() {
   echo -e "${OVER}  ${TICK} DNS resolution is available"
 }
 
+# Function: try_restore_backup
+# Description: Attempts to restore the previous Pi-hole gravity database from a
+#              backup file. If a backup exists, it copies the backup to the
+#              gravity database file and prepares a new gravity database. If the
+#              restoration is successful, it returns 0. Otherwise, it returns 1.
+# Returns:
+#   0 - If the backup is successfully restored.
+#   1 - If no backup is available or if the restoration fails.
+try_restore_backup () {
+  local num
+  num=$1
+  # Check if a backup exists
+  if [ -f "${gravityBCKfile}.${num}" ]; then
+    echo -e "  ${INFO} Attempting to restore previous database from backup"
```

Comment:
Maybe we should move this message to line 428 (before the for loop) to avoid repetition, or change the message to include the backup file number:

```suggestion
    echo -e "  ${INFO} Attempting to restore previous database from Backup no. ${num}"
```

Reply:
Any specific reason for writing `Backup` starting with a capital letter? Not being a native speaker, my feeling is still that nothing is to be written with a capital letter in English mid-sentence - except for names.

Reply:
No reason at all... I just copied/pasted the text from line 382 and didn't notice the uppercase letter.

I just fixed my suggestion.